src/modules/tokenization/components/DisplayCards/OffpixButtonBase/index.ts

```diff
@@ -0,0 +1 @@
+export * from './OffpixButtonBase';
```

Comment:
Isso está exportando apenas as interfaces e types do arquivo, pois o componente em si do arquivo está com export default
O ideal seria padronizar as exportações pra caso o componente precise ser exportado isolado a maneira de importar para quem for consumir a lib seja igual para todos os componentes